cmd/root.go

```diff
@@ -405,31 +400,16 @@ func NewCommand(opts ...Option) *Command {
 	rootCmd.AddCommand(waitCmd)
 
 	rootCmd.Args = func(cmd *cobra.Command, args []string) error {
-		// If args is not already populated, try to read from the environment.
-		if len(args) == 0 {
-			args = instanceFromEnv(args)
-		}
-		// Override CLI based arguments with any programmatic options.
-		for _, o := range opts {
```

Comment:
We need to keep doing this below -- pass opts to loadArgs, and just keep four lines.

Reply:
yes - missed that. Reintroduced it in the loadArgs func 🙂